_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,79 @@
+- [Alerting](https://github.com/opensearch-project/alerting/blob/main/release-notes/opensearch-alerting.release-notes-1.0.0.0-rc1.md)
+- [Alerting Dashboards](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/release-notes/opensearch-alerting-dashboards-plugin.release-notes-1.0.0.0-rc1.md)
+- [SQL/PPL OpenSearch Plugin](https://github.com/opensearch-project/sql/blob/main/release-notes/opensearch-sql.release-notes-1.0.0.0-rc1.md)
+- [Query Workbench Dashboards Plugin](https://github.com/opensearch-project/sql/blob/main/release-notes/opensearch-sql.release-notes-1.0.0.0-rc1.md)
+- [SQL JDBC Driver](https://github.com/opensearch-project/sql/blob/main/release-notes/opensearch-sql.release-notes-1.0.0.0-rc1.md)
+- [SQL CLI Client](https://github.com/opensearch-project/sql/blob/main/release-notes/opensearch-sql.release-notes-1.0.0.0-rc1.md)
+- [SQL ODBC Driver](https://github.com/opensearch-project/sql/blob/main/release-notes/opensearch-sql.release-notes-1.0.0.0-rc1.md)
+- [Notebooks OpenSearch Plugin](https://github.com/opensearch-project/dashboards-notebooks/blob/main/release-notes/opensearch-dashboards-notebooks.release-notes-1.0.0.0-rc1.md)
+- [Notebooks Dashboards Plugin](https://github.com/opensearch-project/dashboards-notebooks/blob/main/release-notes/opensearch-dashboards-notebooks.release-notes-1.0.0.0-rc1.md)
+- [Reporting Opensearch Plugin](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-rc1.md)
+- [Reporting Dashboards Plugin](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-rc1.md)
+- [Trace Analytics Dashboards Plugin](https://github.com/opensearch-project/trace-analytics/blob/main/release-notes/opensearch-trace-analytics.release-notes-1.0.0.0-rc1.md)
+- [Gantt Charts Dashboards Visualization Plugin](https://github.com/opensearch-project/dashboards-visualizations/blob/main/release-notes/opensearch-dashboards-visualizations.release-notes-1.0.0.0-rc1.md)
+- [Async-Search](https://github.com/opensearch-project/asynchronous-search/blob/main/release-notes/opensearch-asynchronous-search.release-notes-1.0.0.0-rc1.md)
+- [Common-Utils](https://github.com/opensearch-project/common-utils/blob/main/release-notes/opensearch-common-utils.release-notes-1.0.0.0-rc1.md)
+
+Consolidated release notes can be found [here](https://github.com/opensearch-project/OpenSearch/tree/main/release-notes). 
+
+**What does this Release Candidate not include?** 
+
+* *Minimum Artifacts* - Based on community [feedback](https://github.com/opensearch-project/opensearch-build/issues/31), we plan on releasing standalone, minimal downloadable artifacts without OpenSearch plugins (alerting, AD, security, etc.). We are expecting to deliver these artifacts by GA.
```

Comment:
replace 'we' where appropriate.

Reply:
I would be a little more crisp with the last sentence. Two things:

1. 'these' is a tad confusing following the list of plugins. Rephrase without the pronoun.
2. Can we just say that they are going to be part of GA instead of 'expecting'? Is there really a question?

Reply:
Done, done, and done.